substrate/frame/bounties/src/tests.rs

```diff
@@ -160,7 +160,7 @@ impl Config<Instance1> for Test {
 	type RuntimeEvent = RuntimeEvent;
 	type BountyDepositBase = ConstU64<80>;
 	type BountyDepositPayoutDelay = ConstU64<3>;
-	type BountyUpdatePeriod = ConstU64<20>;
+	type BountyUpdatePeriod = ();
```

Comment:
Hm, now we can test only one. You could add a `parameter_types!` and then dynamically set that in tests to test both cases.